src/WidgetBase.ts

```diff
@@ -241,6 +259,33 @@ export class WidgetBase<P extends WidgetProperties = WidgetProperties, C extends
 		this._checkOnElementUsage();
 	}
 
+	protected meta<T extends WidgetMeta>(MetaType: WidgetMetaConstructor<T>): T {
+		let cached = this._metaMap.get(MetaType);
+		if (!cached) {
+			const boundInvalidate = this.invalidate.bind(this);
+			const invalidate = () => {
```

Comment:
I know I made this an arrow function, but it could (should?) just be a plain old function